### DISCUSSION TITLE
SET LIMITS FOR AUTOMATION PARAMETERS

### DISCUSSION THREAD

**Discussion by worker-gray** [SynthstromAudible/DelugeFirmware#2925] [Ideas]
Sometimes the variables of an automation screen are just too much. For instance, i have a synth who's pitch bend I don't want to exceed +7 or -3. It would be great if I could 'lock-in' a set range-- either for that instrument or that track

**Comment by seangoodvibes** [comment]
have you tried pad selection mode? you can easily fine tune automation values to what you want.

use shift + waveform while you have a parameter selected to enter pad selection mode

you hold two pads to select start and finish or just a single pad and use the gold knobs to fine tune the values

or even outside pad selection mode, hold a step or multiple steps and turn gold knob.

**Reply by worker-gray** [reply]
Yeah-- that's how I usually do it. I just thought setting automation limits (or setting limits in general) might be convenient and advantageous (not for the coders of course)
